layers/icmp6.go

```diff
@@ -165,9 +165,8 @@ func CreateICMPv6TypeCode(typ uint8, code uint8) ICMPv6TypeCode {
 // ICMPv6 is the layer for IPv6 ICMP packet data
 type ICMPv6 struct {
 	BaseLayer
-	TypeCode  ICMPv6TypeCode
-	Checksum  uint16
-	TypeBytes []byte
```

Comment:
Please don't remove TypeBytes, as that's a backwards-incompatible change which could break code of anyone currently referencing the field.

Reply:
Alright, I'll add it back to the struct. Should I keep the old behavior of setting it to the `data[4:8]` or leave it empty?